dashboard/modules/log/log_manager.py

```diff
@@ -273,21 +298,43 @@ async def resolve_filename(
 
             worker_id = task.get("worker_id", None)
             node_id = task.get("node_id", None)
+            log_info = task.get("task_log_info", None)
```

Comment:
Does it make sense to break down each if to a separate method (if actor_id, elif task_id, elif submission_id, etc.)? Looks like this function is really big now.